src/components/core/PivotTable.tsx

```diff
@@ -260,6 +262,21 @@ export const getSortsFromModel = (
     });
 };
 
+const totalsHasChanged = (gridApi: GridApi, totals: IGridTotalsRow[]) => {
```

Comment:
Move this to some other file so PivotTable.tsx won't pile up more

Reply:
Good idea

Reply:
Moved to separate file along with the whole `getAGGridDataSource` function.